cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/StatefulSetBrokerProvisioner.java

```diff
@@ -42,7 +38,7 @@ public void configure(Map<String, ?> configs) {
     @Override
     protected ProvisionerState addOrRemoveBrokers(ProvisionRecommendation rec) {
 
-        try (final KubernetesClient client = new KubernetesClientBuilder().build()) {
+        try (KubernetesClient client = new KubernetesClientBuilder().build()) {
```

Comment:
why should `KubernetesClient` not be final?

Reply:
I think it should be, but the Cruise Control repo has some pretty opinionated lints and this was one of them.

Reply:
(In general, they don't use `final` in a lot of places that other Java developers might...but I think the "correct" use of `final` is one of those tabs vs spaces type discussions in the Java world)